### ISSUE TITLE
Use the Bayesian model parameters for budget optimization

### ISSUE THREAD

**Issue by shuvayan** [pymc-labs/pymc-marketing#521]
Hello,

Currently we are use michaelis-menten or the sigmoid curves to optimize the budget allocation.
Is there a way to use the model parameters to predict on unseen data.
In simpler terms, I have already built the model to parameterize the effects on adspend and have the bayesian model. Can I put in the new data (future adspend by channel budget) and optimize that using the posterior predictive distributions?

**Comment by shuvayan** [comment]
Hello @cetagostini ,

Thank you for the reply and clear explanation of the problem. I believe you are right, ideally I would like to see if I allocate budget in n different ways across 'x' channels:
1. What would be my predictions of revenue?
2. What would be the response curves/channel contributions in each scenario?
3. What would be the optimal allocation proportions across the 'x' channels according to the bayesian model parameters?

Does that make sense or am I not being lucid enough? :)

BTW, congratulations on the awesome library!!

**Comment by shuvayan** [comment]
Hello @cetagostini ,

If you can guide me on creating this enhancement, I can start and proceed accordingly.
Thank you!

**Comment by wd60622** [comment]
Hi @shuvayan , 

I think the package has evolved a bit since this issue. Do you still have some questions here? Maybe the functionality has become a bit more clear

**Comment by shuvayan** [comment]
I think this is clear now.
